server/helpers/mailing.js

```diff
@@ -1,21 +1,122 @@
-const setUpdateProfileEmailOptions = (adminMail, emailTo, name, lastname, email, newPassword) => {
-  let text = `The data of your profile just was changed.
-        \nYour new data is the next:
-        \nName: ${name}
-        \nLastname: ${lastname}
-        \nEmail: ${email}`;
-  if (newPassword) {
-    text += `\nPassword: ${newPassword}`;
-  }
-  text += '\n\nWarning! If you change your email, new notification will not arive to this email anymore.';
-
-  return {
-    from: `"Admin TripReview" <${adminMail}>`,
-    to: emailTo,
-    subject: 'Update Profile',
-    text,
-  };
-};
+const generateResetPasswordTemplate = (user, password) => `
+  <div style="font-family: sans-serif;">
+    <div style="text-align: center;
+      font-size: 28px;
+      color: #ffffff;
+      border-bottom: 6px solid #0495a8;
+      display: block;
+      padding: 10px;
+      background: #00BCD4;"
+    >
+      Travel Stories
+    </div>
+    <div style="padding: 0 50px; max-width: 700px; margin: auto;">
+      <h2 style="text-align: center; color: black;">Password recovery</h2>
+
+      <p style="color: black;">Dear <b>${user.name} ${user.lastname}</b>!</p>
+      <p style="color: black;">You are receiving this email because you have requested a new password for this email.</p>
+      <p style="color: black;">Your new password is: <b>${password}</b></p>
+      <p style="color: black;">Now you can access your account with the password provided and change it for the new one in the settings of your profile.</p>
+      <p style="color: black;">
+        I would appreciate a lot if you provide your feedback via
+        <a href="https://github.com/KovDimaY/Trip-Reviewer" target="_blank" style="font-weight: 600; color: #02bcd4;">GitHub</a>, 
+        <a href="https://www.linkedin.com/in/kovalenkodmytro" target="_blank" style="font-weight: 600; color: #02bcd4;">LinkedIn</a> or
+        <a href="https://www.facebook.com/dmytro.kovalenko.1004" target="_blank" style="font-weight: 600; color: #02bcd4;">Facebook</a>. 
+      </p>
+      <p style="text-align: center;
+        font-size: 24px;
+        color: black;
+        margin-top: 50px;"
+      >
+        I wish you a good day and thank you for using my project! 😇
+      </p>
+    </div>
+    <div style="padding: 20px;
+      margin-top: 50px;
+      background-color: #f2f2f2;
+      -webkit-box-shadow: 0px -5px 0px #bfbfbf;
```

Comment:
As I know this property doesn't work in the most mail client.
But I am not sure.
You should know that not all css properties work in mail client.
https://templates.mailchimp.com/resources/email-client-css-support/

Reply:
Yes, good point! Thank you! I will change it to border-top... 👍